docs/how-to-docs/keycloak-sidecar.md

```diff
@@ -0,0 +1,109 @@
+# Using Keycloak Sidecar
+
+The keycloak gatekeeper (also know as the keycloak sidecar) is used to enable SSO on applications. [More information on configuration options](https://www.keycloak.org/docs/8.0/securing_apps/#configuration-options)
+
+## Keycloak sidecar using single role
+
+Below is a example of a keycloak sidecar which uses a single role for all URIs.In this example the configuration options are passed in through environment variables and command arguments.
+
+```yaml
+- name: keycloak-gatekeeper
+  image: quay.io/keycloak/keycloak-gatekeeper:latest
+  ports:
+    - containerPort: 10443
+      name: https
+      protocol: TCP
+  env:
+      # the secret of the client you are using. You can find this secret under the credentials tab of the client
+    - name: PROXY_CLIENT_SECRET
+      valueFrom:
+        secretKeyRef:
+          key: client_secret
+          name: gatekeeper
+  args:
+    # specifies what port the keycloak-gatekeeper is listening on
+    - --listen=:10443
+    # specififes which client is used
+    - --client-id=gatekeeper
+    # the url which is used to retrieve the OpenID configuration
+    - --discovery-url=https://sso-dev.notprod.homeoffice.gov.uk/auth/realms/hod-test
+    # the endpoint where requests are proxied to
+    - --upstream-url=https://127.0.0.1:10444
+    # The location of the certificate you wish the proxy to use for TLS support
+    - --tls-cert=/certs/tls.crt
+    # The location of a private key for TLS
+    - --tls-private-key=/certs/tls.key
+    # URls that you wish to protect.
+    - --resources=uri=/*
+  resources:
+    limits:
+      memory: 100Mi
+    requests:
+      memory: 50Mi
+  securityContext:
+    procMount: Default
```

Comment:
we don't need the procMount attribute, do we? it will be the default if not specified?
just keeping about keeping the example short for people to not be distracted by stuff that is not relevant to / required by the keycloak sidecar

Reply:
Removed.